src/main/resources/application.properties

```diff
@@ -16,38 +19,29 @@ rules.replacement.marker: %IdentifyRulesEventId%
 # This property needs to be enabled in order to test new aggregation rules on an
 # existing or artificially created set of events.
 # WARNING! Do not enable this in a production environment!
-testaggregated.enabled: false
-
-# Logging levels can be set with the flag below for the top packages deactivated by
-# default because Travis will fail due to log file size. It is possible to only
-# activate logging for a certain package under these top packages.
-logging.level.root: OFF
-logging.level.org.springframework.web: ERROR
-logging.level.com.ericsson.ei: ERROR
+test.aggregation.enabled: false
```

Comment:
I just realized front-end refers to this property by name so updates are also required in front-end repository.

Reply:
I'll take care of this later.

Reply:
PR is here https://github.com/eiffel-community/eiffel-intelligence-frontend/pull/284